Makefile

```diff
@@ -12,7 +13,7 @@ build:                              ## Build docker image
 	BUILD_ENV=$(BUILD_ENV) $(dc) build --no-cache
 
 start:                              ## Run frontend
-	$(dc) run --service-ports frontend
+	CONFIG=$(CONFIG) npm start
```

Comment:
In my opinion simple npm commands don't belong in this file - especially if the `start` and `stop` commands use different tooling (docker vs npm)

Reply:
Fair enough, will revert this